docs/index.md

```diff
@@ -42,7 +42,7 @@ features:
 /* Homepage Components button */
 .VPContent.is-home .VPButton.medium.alt[href^="/components/"] {
   color: #fff;
```

Comment:
suggestion: use text color var

Reply:
I tried and for some reason `$kui-` SCSS variables don't work in this file. Either way we can tackle this later.